Formula/terraforming.rb

```diff
@@ -0,0 +1,20 @@
+class Terraforming < Formula
+  desc "Export existing AWS resources to Terraform style (tf, tfstate)"
+  homepage "http://terraforming.dtan4.net/"
+  url "https://github.com/dtan4/terraforming.git",
+      :tag => "v0.14.0",
+      :revision => "5e934a1a8d18062f9c4e4e3f4a52dd648fc9976d"
+  head "https://github.com/dtan4/terraforming.git"
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    system "gem", "build", "terraforming.gemspec"
+    system "gem", "install", "terraforming-0.14.0.gem"
```

Comment:
The gem dependencies need to be vendored into the formula. See `braid`, `ghi`, `haste-client`, and `mikutter` for examples.